Tests/CertLogicTests/RuleEngineTests.swift

```diff
@@ -475,8 +475,8 @@ final class RuleEngineTests: XCTestCase {
                                                                         {"ver":"1.0.0"}
                                                                   """)
         
-        XCTAssertTrue(result[0].result == .passed)
```

Comment:
Why did the order change of the results?

Reply:
i have reverted that.
The order in the array is "random"